internal/cmd/gocdk/internal/static/_assets/demo/demo_pubsub.go

```diff
@@ -32,8 +35,11 @@ var subscription *pubsub.Subscription
 var subscriptionErr error
 
 func init() {
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-	defer cancel()
+	// TODO(rvangent): This breaks due to the oauth2 library capturing the ctx:
```

Comment:
Unfortunately, this comment is going to be user-visible, too. Optionally, you might want to remove this in favor of the issue.

Reply:
Done.